sdk/physical/cache.go

```diff
@@ -15,9 +15,9 @@ const (
 	// DefaultCacheSize is used if no cache size is specified for NewCache
 	DefaultCacheSize = 128 * 1024
 
-	// refreshCacheCtxKey is a ctx value that denotes the cache should be
+	// RefreshCacheCtxKey is a ctx value that denotes the cache should be
 	// refreshed during a Get call.
-	refreshCacheCtxKey = "refresh_cache"
```

Comment:
Can we use CacheRefreshContext(ctx,true) Function rather than exposing this constant ?

Reply:
yes. we can. will update this.